src/app/donation-start/donation-start-container/donation-start-container.component.html

```diff
@@ -32,10 +32,12 @@ <h1 class="b-rh-2 b-bold">{{ campaign.title }}</h1>
         <span class="b-rt-1 b-bold">{{ reservationExpiryDate | timeLeft }}</span>
       </p>
 
-      <app-donation-start-form *ngIf="campaignOpenOnLoad"
+      <!-- We don't want to let the form's ngInit run before ours completes, or we would have odd behaviour
+           around loading the logged in user - hence conditional on dataLoaded -->
+      <app-donation-start-form *ngIf="dataLoaded && campaignOpenOnLoad"
```

Comment:
I'm not actually 100% sure if this change is needed - yesterday it seemed like it was needed, in my testing now on my local I wasn't able to keep reproduce the bug just by changing this line back. But I think its worth keeping anyway, easier to see what's going on if the form doesn't load until the onnginit from the container is finished.